mabl snippets/native mobile testing/mobileZoomInAtScreenCenter.js

```diff
@@ -0,0 +1,71 @@
+function mablJavaScriptStep(mablInputs, callback) {
+  async function callAppium() {
+    const driver = await mabl.mobile.getDriver();
+    // Find the element by xpath
+    const target = await driver.findElement("xpath", "//*");
+    // Get the rect of the element
+    const center = await getElementCenterXY(target);
+    // Define the actions to perform
+    const actions = [
+      {
+        type: "pointer",
+        id: "leftFinger",
+        parameters: { pointerType: "touch" },
+        actions: [
+          // Move the pointer to the left of the center
+          { type: "pointerMove", duration: 0, x: center.x * 0.45, y: center.y },
+          // Press down
+          { type: "pointerDown", button: 0 },
+          // Move the pointer further to the center over 1000ms
+          {
+            type: "pointerMove",
+            duration: 1000,
+            x: center.x * 0.35,
```

Comment:
Based on the percentages used here, this is actually moving away from the center I think?